en/airframes/adding_a_new_frame.md

```diff
@@ -197,17 +197,20 @@ The following *PX4 User Guide* topics explain how to tune the parameters that wi
 * [VTOL Configuration](https://docs.px4.io/en/config_vtol/)
 
 
+## Add New Airframe to QGroundControl
 
-## Getting a New Airframe to Show in QGroundControl
+To make a new airframe available for section in the *QGroundControl* [airframe configuration](https://docs.px4.io/en/config/airframe.html):
 
-* After a clean build (e.g. by running `make clean` and then `make px4fmu-v5_default upload`), open QGC and flash a "Custom firmware file..." as shown below:
+1. Make a clean build (e.g. by running `make clean` and then `make px4fmu-v5_default upload`)
```

Comment:
If you're using QGC to upload, you don't need the `upload` in the `make` command, right?

Reply:
Very good point! I will fix.